code/tsconfig.json

```diff
@@ -24,5 +24,11 @@
     "compilerOptions": {
       "types": ["node"]
     }
+  },
+  "angularCompilerOptions": {
+    "enableI18nLegacyMessageIdFormat": false,
+    "strictInjectionParameters": true,
+    "strictTemplates": true
+    // ...
```

Comment:
is this needed?

Reply:
Thank you for catching this, I removed the accidentally added settings.